docs/core/deploying/index.md

```diff
@@ -271,6 +288,49 @@ When you deploy your application, any third-party dependencies used in your app
 
 Note that you can only deploy a self-contained app with a third-party library to platforms supported by that library.
 
+## Deploying a Modular Self-Contained App ##
+
+The previous two sections on deploying a self-contained app created and deployed an app that contained the entire .NET Core library. You can also deploy a modular self-contained app that includes only the .NET Core assemblies actually needed by your app.
+
+The word parsing example shown earlier in the [Deploying a Simple Self-Contained App](#simpleSelf) section uses the console, which is defined in the System.Console assembly, and the regular expression engine, which is defined in the System.Text.RegularExpressions assembly. Instead of deploying all of the .NET Standard Library with our app, we can deploy just the assemblies we need, the dependencies of those assemblies, and the libraries that contain basic .NET Core types. This requires a modification of the `dependencies` section of our project.json file:
+
+```json
+  "dependencies": {
+    "System.Console": "4.0.0",
+    "System.Text.RegularExpressions": "4.1.0",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "Microsoft.NETCore.DotNetHostPolicy": "1.0.1"
+  },
+```
+
+For more information, see [Reducing Package Dependencies with project.json](reducing-dependencies.md).
```

Comment:
This is no longer something we want to emphasize.  The story around dependencies and NuGet isn't complete yet.  I'd remove this line.